tasks/page/processBOM.js

```diff
@@ -94,7 +96,15 @@ if (require.main !== module) {
     kitnicYaml = {}
   }
   if (kitnicYaml.multi) {
-    kitnicYaml = kitnicYaml.multi[projectPath]
+    const projects = Object.keys(kitnicYaml.multi)
+    const multiProjectPath = projects.find(project => {
+      return (
+        kitnicYaml.multi[project].path === projectPath ||
+        project === projectPath
+      )
+    })
```

Comment:
Move this out to a helper function? Would it make sense to place it in the `utils.js` file?

Reply:
I guess so. It might not be worth it just on it's own.

It would be really nice to refactor all of the tasks in a way that they can later be re-used when we manage to move away from using ninja to build the site i.e. allow us to build a page dynamically when a user uploads their files. That's a much bigger task though.

Reply:
Would be willing to (slowly) work on that.

For now, I moved this logic into the `utils.js` file.